data/json/items/armor/helmets.json

```
@@ -771,6 +771,7 @@
     "color": "light_gray",
     "warmth": 5,
     "material_thickness": 2,
+    "qualities": [ [ "COOK", 3 ], [ "BOIL", 2 ], [ "CONTAIN", 1 ], [ "CHEM", 1 ] ],
     "techniques": [ "WBLOCK_1" ],
```

```suggestion
    "copy-from": "pot_helmet",
```
The "looks_like" and "symbol" could also be removed.